File: Day-08/Pattern8.js
function patternEight(n) {
  let i = 1;
  let count = 1;
  while (i <= n) {
    let j = 1;
    let str = "";
    while (j <= i) {
      str += count;
      count++;
      j++;
    }
    i++;
    console.log(str);
  }
}

patternEight(4);

File: Day-08/Pattern8.py
n = int(input())

def PatternEight(n):
    i = 1
    count = 1
    while(i <=n):
        j = 1
        while(j <= i):
            print(count ,end='')
            j = j +1
            count = count +1
        print(end='\n')
        i = i + 1
    
PatternEight(n)